tsl/test/expected/continuous_aggs_ddl-10.out

```diff
@@ -798,7 +800,7 @@ SELECT * FROM drop_chunks_view ORDER BY time_bucket DESC;
           10 |  14
            5 |   9
            0 |   4
-(8 rows)
+(10 rows)
```

Comment:
why did this output change given that nothing else has (should have) changed (modulo the additional view creation param to get the existing/old behavior)?

Reply:
that cagg didnt get its parameters changed and was already using the new behaviour